lib/Service/ColumnTypes/DatetimeBusiness.php

```diff
@@ -9,9 +9,11 @@
 class DatetimeBusiness extends SuperBusiness implements IColumnTypeBusiness {
 
 	/**
-	 * @throws Exception
+	 * @param mixed $value (string|null)
+	 * @param Column|null $column
+	 * @return string
 	 */
-	public function parseValue(string $value, ?Column $column = null): string {
+	public function parseValue($value, ?Column $column = null): string {
```

Comment:
```suggestion
	public function parseValue(?string $value, ?Column $column = null): string {
```

Why not this?

Reply:
Another note, the method itself does no null check, so it would still be passed to DateTime would cause a deprecation warning from PHP, so better catch that earlier and return.

Reply:
Ok, then let's keep the mixed type but properly check that it is a string and otherwise return early.